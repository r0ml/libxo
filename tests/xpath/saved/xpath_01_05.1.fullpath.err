main: input '#run --libxo @fullpath:slax,pretty'
main: input '#run'
main: input '?one/two/three/four'
--- 0: 9
    000009: type 74 (C_PATH), str 0 [], contents 1, next 0, prev 0 
        000001: type 70 (C_ELEMENT), str 1 [one], contents 0, next 3, prev 0 
        000003: type 70 (C_ELEMENT), str 7 [two], contents 0, next 5, prev 1 
        000005: type 70 (C_ELEMENT), str 13 [three], contents 0, next 7, prev 3 
        000007: type 70 (C_ELEMENT), str 21 [four], contents 0, next 0, prev 5 
main: status: zero
main: input '?a/b/c/d'
--- 0: 9
    000009: type 74 (C_PATH), str 0 [], contents 1, next 0, prev 0 
        000001: type 70 (C_ELEMENT), str 1 [one], contents 0, next 3, prev 0 
        000003: type 70 (C_ELEMENT), str 7 [two], contents 0, next 5, prev 1 
        000005: type 70 (C_ELEMENT), str 13 [three], contents 0, next 7, prev 3 
        000007: type 70 (C_ELEMENT), str 21 [four], contents 0, next 0, prev 5 
--- 1: 18
    000018: type 74 (C_PATH), str 0 [], contents 10, next 0, prev 0 
        000010: type 70 (C_ELEMENT), str 26 [a], contents 0, next 12, prev 0 
        000012: type 70 (C_ELEMENT), str 30 [b], contents 0, next 14, prev 10 
        000014: type 70 (C_ELEMENT), str 34 [c], contents 0, next 16, prev 12 
        000016: type 70 (C_ELEMENT), str 38 [d], contents 0, next 0, prev 14 
main: status: zero
main: input '?!e'
--- 0: 9
    000009: type 74 (C_PATH), str 0 [], contents 1, next 0, prev 0 
        000001: type 70 (C_ELEMENT), str 1 [one], contents 0, next 3, prev 0 
        000003: type 70 (C_ELEMENT), str 7 [two], contents 0, next 5, prev 1 
        000005: type 70 (C_ELEMENT), str 13 [three], contents 0, next 7, prev 3 
        000007: type 70 (C_ELEMENT), str 21 [four], contents 0, next 0, prev 5 
--- 1: 18
    000018: type 74 (C_PATH), str 0 [], contents 10, next 0, prev 0 
        000010: type 70 (C_ELEMENT), str 26 [a], contents 0, next 12, prev 0 
        000012: type 70 (C_ELEMENT), str 30 [b], contents 0, next 14, prev 10 
        000014: type 70 (C_ELEMENT), str 34 [c], contents 0, next 16, prev 12 
        000016: type 70 (C_ELEMENT), str 38 [d], contents 0, next 0, prev 14 
--- 2: 19
    000019: type 73 (C_NOT), str 40 [!], contents 20, next 0, prev 0 
        000020: type 70 (C_ELEMENT), str 42 [e], contents 0, next 0, prev 0 
main: status: zero
main: input '?uno/dos/tres/cuatro'
--- 0: 9
    000009: type 74 (C_PATH), str 0 [], contents 1, next 0, prev 0 
        000001: type 70 (C_ELEMENT), str 1 [one], contents 0, next 3, prev 0 
        000003: type 70 (C_ELEMENT), str 7 [two], contents 0, next 5, prev 1 
        000005: type 70 (C_ELEMENT), str 13 [three], contents 0, next 7, prev 3 
        000007: type 70 (C_ELEMENT), str 21 [four], contents 0, next 0, prev 5 
--- 1: 18
    000018: type 74 (C_PATH), str 0 [], contents 10, next 0, prev 0 
        000010: type 70 (C_ELEMENT), str 26 [a], contents 0, next 12, prev 0 
        000012: type 70 (C_ELEMENT), str 30 [b], contents 0, next 14, prev 10 
        000014: type 70 (C_ELEMENT), str 34 [c], contents 0, next 16, prev 12 
        000016: type 70 (C_ELEMENT), str 38 [d], contents 0, next 0, prev 14 
--- 2: 19
    000019: type 73 (C_NOT), str 40 [!], contents 20, next 0, prev 0 
        000020: type 70 (C_ELEMENT), str 42 [e], contents 0, next 0, prev 0 
--- 3: 30
    000030: type 74 (C_PATH), str 0 [], contents 22, next 0, prev 0 
        000022: type 70 (C_ELEMENT), str 44 [uno], contents 0, next 24, prev 0 
        000024: type 70 (C_ELEMENT), str 50 [dos], contents 0, next 26, prev 22 
        000026: type 70 (C_ELEMENT), str 56 [tres], contents 0, next 28, prev 24 
        000028: type 70 (C_ELEMENT), str 63 [cuatro], contents 0, next 0, prev 26 
main: status: zero
main: input '+one'
main: status: track
main: input '+two'
main: status: track
main: input '+three'
main: status: track
main: input '+four'
main: status: full
main: input '=data foo'
main: field: 'data'='foo'
main: status: full
main: input '+five'
main: status: full
main: input '=data goo'
main: field: 'data'='goo'
main: status: full
main: input '+six'
main: status: full
main: input '=data hoo'
main: field: 'data'='hoo'
main: status: full
main: input '-six'
main: status: full
main: input '-five'
main: status: full
main: input '=data ioo'
main: field: 'data'='ioo'
main: status: full
main: input '-four'
main: status: track
main: input '=data joo'
main: field: 'data'='joo'
main: status: track
main: input '-three'
main: status: track
main: input '-two'
main: status: track
main: input '+two'
main: status: track
main: input '+a'
main: status: track
main: input '+b'
main: status: track
main: input '+c'
main: status: track
main: input '+uno'
main: status: track
main: input '-uno'
main: status: track
main: input '+uno'
main: status: track
main: input '-uno'
main: status: track
main: input '+d'
main: status: full
main: input '+e'
main: status: track
main: input '+f'
main: status: track
main: input '=g h'
main: field: 'g'='h'
main: status: track
main: input '-f'
main: status: track
main: input '-e'
main: status: full
main: input '-d'
main: status: track
main: input '+d'
main: status: full
main: input '=data koo'
main: field: 'data'='koo'
main: status: full
main: input '+e'
main: status: track
main: input '+f'
main: status: track
main: input '=data loo'
main: field: 'data'='loo'
main: status: track
main: input '-f'
main: status: track
main: input '-e'
main: status: full
main: input '-d'
main: status: track
main: input '-c'
main: status: track
main: input '-b'
main: status: track
main: input '-a'
main: status: track
main: input '+three'
main: status: track
main: input '+four'
main: status: full
main: input '=data moo'
main: field: 'data'='moo'
main: status: full
main: input '-four'
main: status: track
main: input '-three'
main: status: track
main: input '-two'
main: status: track
main: input '-one'
main: status: track
