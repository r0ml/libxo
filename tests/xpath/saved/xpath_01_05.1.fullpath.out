/one/two/three/four/data = 'foo'
/one/two/three/four/five/data = 'goo'
/one/two/three/four/five/six/data = 'hoo'
/one/two/three/four/data = 'ioo'
/one/two/a/b/c/d/data = 'koo'
/one/two/three/four/data = 'moo'
